tests/unit_tests/test_wave_06.py

```diff
@@ -228,16 +234,18 @@ def test_swap_best_by_category_no_match_is_false():
         their_priority="Clothing"
     )
 
-    raise Exception("Complete this test according to comments below.")
-    # *********************************************************************
-    # ****** Complete Assert Portion of this test **********
-    # *********************************************************************
-    # Assertions should check:
-    # - That result is falsy
-    # - That tai and jesse's inventories are the correct length
-    # - That all the correct items are in tai and jesse's inventories
+    assert len(jesse.inventory) == len(tai.inventory)
+    assert item_d not in tai.inventory
+    assert item_d in jesse.inventory
+    assert item_f in jesse.inventory
+    assert item_f not in tai.inventory
+    assert item_a in tai.inventory
+    assert item_a not in jesse.inventory
+    assert item_c in tai.inventory
+    assert item_c not in jesse.inventory
+    assert result == False
```

Comment:
👍 nice job explicitly checking that `result` is `False` and not just a falsey value.

Reply:
✅ Thanks!